tests/integration/update_cluster/minimal_scaleway/kubernetes.tf

```diff
@@ -212,24 +227,29 @@ resource "scaleway_instance_volume" "etcd-1-etcd-main-scw-minimal-k8s-local" {
 }
 
 resource "scaleway_lb" "api-scw-minimal-k8s-local" {
-  ip_id = scaleway_lb_ip.api-scw-minimal-k8s-local.id
-  name  = "api.scw-minimal.k8s.local"
-  tags  = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local", "noprefix=kops.k8s.io/role=ControlPlane"]
-  type  = "LB-S"
+  description = "Load-balancer for kops cluster scw-minimal.k8s.local"
+  ip_id       = scaleway_lb_ip.api-scw-minimal-k8s-local.id
+  name        = "api.scw-minimal.k8s.local"
+  tags        = ["noprefix=kops.k8s.io/cluster=scw-minimal.k8s.local", "noprefix=kops.k8s.io/role=ControlPlane"]
+  type        = "LB-S"
```

Comment:
Could this be put in a variable to be changed quickly in case the default type of LB would change name?

Reply:
Same here